cpp/src/cider/exec/module/batch/CiderArrowBufferHolder.h

```diff
@@ -71,6 +71,9 @@ class CiderArrowSchemaBufferHolder {
 
   ArrowSchema* getDictPtr();
 
+  // Buffer required to generate a decimal format.
+  std::string formatBuffer;
```

Comment:
`format_buffer` better?

Reply:
Is this variable actually used? I see you set it but doesn't use it.

Reply:
just workaround for constructing decimal format string. Arrow schema format string is `const char*` type. decimal constructed format str will be destroyed once leave the function, so keep str memory in schemaHolder for workaround.

https://github.com/intel/BDTK/blob/d0f8adfc54bb155bd9b09eab4effde8cd1f50bdb/cpp/src/cider/exec/module/batch/CiderBatchUtils.cpp#L301-L309